### ISSUE TITLE
Is there a way to get all of the addresses that a user owns?

### ISSUE THREAD

**Issue by BlakeBrown** [Berry-Pool/nami-wallet#152]
Just a general question :)

**Comment by alessandrokonrad** [comment]
You mean addresses from the other accounts in Nami?

**Comment by BlakeBrown** [comment]
> You mean addresses from the other accounts in Nami?

Yes, we're looking to show the user all of their wallet addresses in one interface (along with the names of each wallet)? Would be great too if there was someway to verify this on the backend so we can store a list of verified addresses for each user.
